win/pkgs/benchmark.ps1

```diff
@@ -0,0 +1,60 @@
+#Requires -RunAsAdministrator
```

Comment:
The nuget is missing is because this file is built as a static library, it is only consumed while gRPC is building itself, after the compilation is done no one needs to use it.